docusaurus.config.js

```diff
@@ -50,6 +50,12 @@ module.exports = {
         // },
       ],
     },
+    docs: { 
+      sidebar: {
+        autoCollapseCategories: true, 
```

Comment:
we can have it enabled for now, we may want to turn it off in the future.

Reply:
yup sure. its just experimentation for the time being